speech/unit_tests/test_client.py

```diff
@@ -192,45 +214,48 @@ def test_sync_recognize_with_empty_results_no_gax(self):
 
     def test_sync_recognize_with_empty_results_gax(self):
         from google.cloud._testing import _Monkey
-        from google.cloud.speech import _gax as MUT
+
         from google.cloud import speech
+        from google.cloud.speech import _gax as MUT
         from google.cloud.speech.sample import Sample
 
         credentials = _Credentials()
         client = self._makeOne(credentials=credentials, use_gax=True)
         client.connection = _Connection()
+        client.connection.credentials = credentials
+
+        _MockGAPICSpeechAPI._response = self._make_sync_response()
```

Comment:
Don't use a mutable global class here. You can monkey-patch `SpeechApi` with a function instead of a class, and that function could spit out an instance of your `_MockGAPICSpeechAPI` class that has `_response` equal to  `self._make_sync_response()`.